### PR TITLE
Add supported Jobs/workloads by scale-ci to the readme

### PR DESCRIPTION
**Pull request by chaitanyaenr**
This commit also adds info about the OpenShift components being scale
tested and the OpenShift release versions supported.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,12 +20,30 @@ The parameters/configuration of each Job in the scale-ci-pipeline is supplied th
...
+Master Vertical | Control plane density | Creates bunch of objects to stress ApiServer, Etcd and Controller | In progress | :heavy_check_mark: | :heavy_check_mark: |  
+Networking | SDN | uperf benchmarks testing node to node, pod to pod, and svc to svc throughput and latency for TCP and UDP protocols | In progress | :heavy_check_mark: | :heavy_check_mark: |  
+Prometheus | Monitoring | prometheus density focused test that creates pods, and nodes and some workloads especially for openshift-monitoring | In progress | :heavy_check_mark: | :heavy_check_mark: |  
+HTTP | Router | Data-plane workload generator that runs http requests through HAProxy into deployed pods | In progress | :heavy_check_mark: | :heavy_check_mark: |  
+Pgbench | Storage | Checks how well Postgresql performs with RHOCS for glusterfs and gluster-block storage backend | In progress | :heavy_check_mark: | :heavy_check_mark: |  
+Mongodb | Storage | Checks how MongoDB performs with RHOCS for glusterfs and gluster-block storage backend | In progess | :heavy_check_mark: | :heavy_check_mark: |
```
Comment (jmencak) [inline README.md]: Mongodb -> MongoDB?

Comment (chaitanyaenr) [reply README.md]: Fixed, thanks.

**Comment by akrzos** [comment]
LGTM

**Comment by chaitanyaenr** [comment]
@akrzos can we merge this PR?

**Pull request merged**